spec/rails_helper.rb

```diff
@@ -203,24 +203,24 @@ def seed_base_data_for_tests
     Geocoder.configure(lookup: :test)
 
     ["1500 Remount Road, Front Royal, VA 22630",
-     "123 Donation Site Way",
-     "Smithsonian Conservation Center new"].each do |address|
-       Geocoder::Lookup::Test.add_stub(
-         address, [
-           {
-             "latitude" => 40.7143528,
-             "longitude" => -74.0059731,
-             "address" => "1500 Remount Road, Front Royal, VA",
-             "state" => "Virginia",
-             "state_code" => "VA",
-             "country" => "United States",
-             "country_code" => "US"
-           }
-         ]
-       )
-     end
-
-    seed_base_data_for_tests
+       "123 Donation Site Way",
+       "Smithsonian Conservation Center new"].each do |address|
+         Geocoder::Lookup::Test.add_stub(
+           address, [
+             {
+               "latitude" => 40.7143528,
+               "longitude" => -74.0059731,
+               "address" => "1500 Remount Road, Front Royal, VA",
+               "state" => "Virginia",
+               "state_code" => "VA",
+               "country" => "United States",
+               "country_code" => "US"
+             }
+           ]
+         )
+       end
+
+    seed_base_data_for_tests if !ENV["SKIP_SEED"]
```

Comment:
Not sure this is ideal? I could not find another way to skip the suite level tests since those run before tags are evaluated.

Reply:
Um... I'm not sure we should be doing _anything_ at the suite level. 

I think the original idea here was that there would be some suite-level data that we'd add once and not have to add again. Obviously that didn't (and probably couldn't ever) work, so we also added it to every example. My hunch is if you remove this line entirely, all the existing tests would still work.